.github/workflows/deploy.yml

```diff
@@ -50,7 +52,7 @@ jobs:
         run: |
           echo "PATH=$(pwd)/node_modules/.bin/:$PATH" >> $GITHUB_ENV
       - name: run unit tests
-        run: ./test-unit.sh {{ env.STAGE_PREFIX }}{{ env.branch_name }}
+        run: ./test-unit.sh
```

Comment:
This is a unit test fix, covered by another ticket, and I believe it will be merged prior to this PR.  So I won't bother doc'ing it here until needed.